om-2.0.rdf

```diff
@@ -28974,11 +28997,12 @@
   <!-- Unit of Measure Ontologies -->
 
   <!-- Contents -->
-
+  <!-- Particles Ontology -->

```

Comment:
singular, please (i.e. particle, not particles)

Reply:
Changed to ParticleNumber